src/visualization/types.jl

```diff
@@ -273,6 +282,18 @@ returns a `DiffEqBase.ODESolution`) or Trixi's own `solve!` (which returns a
 """
 PlotData2D(sol::TrixiODESolution; kwargs...) = PlotData2D(sol.u[end], sol.prob.p; kwargs...)
 
+# Also redirect when using PlotData2DTriangulate.
+PlotData2DTriangulated(sol::TrixiODESolution; kwargs...) = PlotData2DTriangulated(sol.u[end], sol.prob.p; kwargs...)
+
+# Create a PlotData2DCartesian if the mesh is a TreeMesh; else create a PlotData2DTriangulated.
+function PlotData2D(u, mesh, equations, solver, cache; kwargs...)
```

Comment:
Since you specialize for `mesh::TreeMesh`, I don't think you actually need `if get_name(mesh)=="TreeMesh"...` here. 

Does this work if you just have `PlotData2D(...)` forward to `PlotDataTriangulated2D(...)`?

Reply:
Thanks for reviewing this.
You are right, that works and in c07e8461bb17cdf5b6359f5b4a82be6ec14d5aae I implemented this to get rid of the function above.